browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -285,6 +292,20 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		return false;
 	},
 
+	_preventNonNumericalInput: function(e) {
+		e = e || window.event;
+		var charCode = (typeof e.which == 'undefined') ? e.keyCode : e.which;
+		var charStr = String.fromCharCode(charCode);
+		if (charStr.match(/^[0-9]+$/))
+			return;
+
+		for (var part in this._numberParts) {
```

Comment:
this is not enough, we can still type:
'1.1,,,,,,' or '-13-----'

Best would be to use single regexp where we can use minus sign only on the beginning and separators at correct positions. This regex should check input field value, not only current key code

Reply:
oh, sound good, thanks